netlify/netlify-plugin-nextjs
# feat: support custom response headers on images served via IPX

### Summary

Adds support for custom response headers to be added to images served with IPX

### Test plan

1. Visit [this page](https://deploy-preview-1515--netlify-plugin-nextjs-demo.netlify.app/old/image/)
2. Inspect the requests for images in the developer tools console (network tab, look for the ones that begin with `?url`)
3. should see 'x-test: foobar' in the response headers

### Relevant links (GitHub issues, Notion docs, etc.) or a picture of cute animal

https://github.com/netlify/netlify-ipx/pull/51
https://github.com/netlify/pod-ecosystem-frameworks/issues/148

![IMG-20220810-WA0005](https://user-images.githubusercontent.com/5655473/184164130-bcbcc8bd-44c6-45b9-877c-5ab91d567ee1.jpeg)


### Standard checks:


---

🧪 Once merged, make sure to update the version if needed and that it was published correctly.
